zerver/models/realms.py

```diff
@@ -100,6 +100,88 @@ class OrgTypeDict(TypedDict):
     onboarding_zulip_guide_url: Optional[str]
 
 
+class CommonPolicyEnum(IntEnum):
+    MEMBERS_ONLY = 1
```

Comment:
`OrgTypeEnum` does not use all caps in fields but we do use all caps in some other cases, so there is no consistent apporach that we use currently.

Reply:
Anyway, most of these will be removed once we migrate the settings to groups framework.

Reply:
I like the all-caps pattern for constants, so no reason to change that for existing ones.